lib/fog/vsphere/models/compute/rules.rb

```diff
@@ -0,0 +1,28 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Rules < Fog::Collection
+        autoload :Rule, File.expand_path('../rule', __FILE__)
+        
+        model Fog::Compute::Vsphere::Rule
+        attr_accessor :datacenter
+        attr_accessor :cluster
```

Comment:
Does that work for collection types? All the other collection types in fog-vsphere use `attr_accessor` and this is copied (primarily) from the `servers` type.